apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -424,6 +423,28 @@ def _get_platform_version(self, zephyr_base: str) -> float:
 
         return float(f"{version_major}.{version_minor}")
 
+    def _load_cmsis(self, lib_path: Union[str, pathlib.Path]):
+        """Copy CMSIS header files to generated project."""
+
+        cmsis_path = pathlib.Path(os.environ["CMSIS_PATH"])
```

Comment:
why not make this a project option for now?

Reply:
I think that should be a project options that I missed. Done!